app/src/org/commcare/android/adapters/EntityListAdapter.java

```diff
@@ -203,13 +212,14 @@ public int getItemViewType(int position) {
      */
     @Override
     public View getView(int position, View convertView, ViewGroup parent) {
-        if (actionEnabled && position == actionPosition) {
+        if (actionsCount > 0 && position >= actionsStartPosition) {
```

Comment:
Might be nice to factor these 2 cases out into their own methods, e.g. `getActionView()` and `getNormalView()`